lib/operations/map_reduce.js

```diff
@@ -21,7 +21,8 @@ const exclusionList = [
   'w',
   'wtimeout',
   'j',
-  'writeConcern'
+  'writeConcern',
+  'scope'
```

Comment:
Let's leave a comment just for this line so that future "us" don't waste a day trying to debug what's going on here :see_no_evil: